tools/log-collector/helper.go

```diff
@@ -208,13 +212,12 @@ func getResourceByName(gVResources []apiv1.APIResource, name string) (matchedRes
 // getContainerStatusValue returns whether current and previous container present to capture logs
 func getContainerStatusValue(containerStatus *corev1.ContainerStatus) (conStatObj containerStat) {
 
-	lastState := containerStatus.LastTerminationState
 	currentState := containerStatus.State
 
-	if lastState.Terminated != nil {
+	if currentState.Running != nil && currentState.Terminated != nil {
```

Comment:
I think, this condition will never be `true` as `currentState` cannot be both `Running` & `Terminated`. Check it once.

Reply:
done